childchain/lib/block.js

```diff
@@ -60,17 +66,23 @@ class Block {
 
   createTXOProof(txo) {
     const tree = this.createTree();
-    return txo.value.map((uid) => tree.proof(uid)).reduce((acc, p) => {
-      return Buffer.concat(acc.concat([p]))
+    return txo.value.map(({start, end}) => {
+      const slot = start.div(CHUNK_SIZE).integerValue(BigNumber.ROUND_FLOOR).toNumber();
+      return tree.proof(slot);
```

Comment:
```suggestion
      const slotEnd = end.div(CHUNK_SIZE).integerValue(BigNumber.ROUND_FLOOR).toNumber();
      return [tree.proof(slot), tree.proof(slotEnd)];
```

Proof duplication must be optimized